<!DOCTYPE html>
<html>
  <head>
    <meta name="pypi:repository-version" content="1.1">
    <title>Links for odoo14-addon-sale-automatic-workflow-ignore-exception</title>
  </head>
  <body>
    <h1>Links for odoo14-addon-sale-automatic-workflow-ignore-exception</h1>
    <a href="https://files.pythonhosted.org/packages/32/70/7f514752cf889ea1962d04c6bcf6702803f3efba695547b5ffbfc46ef756/odoo14_addon_sale_automatic_workflow_ignore_exception-14.0.1.0.1-py3-none-any.whl#sha256=d13ea2d82c8da5cc55ceccbe0656a6f03b87b273600c4ead3994ce0451b1964f" data-requires-python="&gt;=3.6" >odoo14_addon_sale_automatic_workflow_ignore_exception-14.0.1.0.1-py3-none-any.whl</a><br />
    </body>
</html>
<!--SERIAL 15264686-->